src/AzureClient/AzureClient.cs

```diff
@@ -522,6 +523,18 @@ public async Task<ExecutionResult> SetActiveTargetAsync(IChannel channel, string
             return AvailableTargets.First(target => target.TargetId == ActiveTarget.TargetId).ToExecutionResult();
         }
 
+        private static IEnumerable<string> ReadAllLines(Stream stream)
```

Comment:
If you are doing encapsulation into `ParseSimulatorOutput` as I suggested in this [comment](https://github.com/microsoft/iqsharp/pull/623/files#r837900742), I would also implement this method inside of `ParseSimulatorOutput` since it is implemented specifically for that and it is short enough.

Reply:
I merged the content of that function into `ParseSimulatorOutput` rather than having a local function.